home-work-app/src/app/pages/movie/components/Description/Description.tsx

```diff
@@ -1,7 +1,59 @@
-import { StyledDescription } from './StyledDescription.styled';
+import { MovieDetails } from './MovieDetails';
+import { useParams } from 'react-router-dom';
+import { IApiResponse } from '../../../../interfaces/IApiResponse';
+import { movieResponse } from '../../../../../../tempData/getMoviesResponse';
+import { find } from 'lodash';
+import { MovieDetailsPoster } from './MovieDetailsPoster.styled';
+import { MovieDetailsContent } from './MovieDetailsContent.styled';
+import { MovieDetailsHeader } from './MovieDetailsHeader';
+import { VoteAverage } from './VoteAverage.styled';
+import { MovieDetailsTitle } from './MovieDetailsTitle.styled';
+import { MovieDetailsTagline } from './MovieDetailsTagline.styled';
+import { MovieDetailsTimeData } from './MovieDetailsTimeData.styled';
+import { MovieDetailsOverview } from './MovieDetailsOverview.styled';
 
-export const Description: React.FC = () => (
-    <StyledDescription>
-        Movie page
-    </StyledDescription>
-);
+const movieCollection: IApiResponse.IMovie[] = movieResponse.data;
+
+export const Description: React.FC = () => {
+    const {id} = useParams();
+    const movie: IApiResponse.IMovie = find(movieCollection, {id: Number(id)});
```

Comment:
For now it's ok but that later that should be replaced with `useEffect` that will load video details.